runtime/src/main/java/org/eclipse/xpanse/runtime/XpanseApplication.java

```diff
@@ -21,6 +22,7 @@
 @EnableJpaRepositories(basePackages = "org.eclipse.xpanse")
 @EntityScan(basePackages = "org.eclipse.xpanse")
 @ComponentScan(basePackages = "org.eclipse.xpanse")
+@EnableJpaAuditing
```

Comment:
do we need this?

Reply:
It is used  to automatically assign values to database column attributes, such as some creation time and modification time